### PR TITLE
TOMEE-2870 Translate to Portuguese: examples/mp-metrics-timed

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira : https://issues.apache.org/jira/browse/TOMEE-2870

### CODE REVIEW

File: examples/mp-metrics-timed/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+application:weather_day_status_seconds{quantile="0.98"} 48352.0
+# TYPE application:weather_day_status_seconds timer
+application:weather_day_status_seconds{quantile="0.99"} 48352.0
+# TYPE application:weather_day_status_seconds timer
+application:weather_day_status_seconds{quantile="0.999"} 48352.0
+----
+
+=== Formato JSON:
+
+Para o formato json, adicione o cabeçalho `Accept: application/json` ao diretório Solicitação HTTP.
```
Comment (willesreis) [inline examples/mp-metrics-timed/README_pt.adoc:121]: Here, it is different of English version.
```suggestion
Para o formato json, adicione o cabeçalho `Accept: application/json` na requisição.
```

File: examples/mp-metrics-timed/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        "p95": 48352,
+        "p98": 48352,
+        "p99": 48352,
+        "p999": 48352,
+        "stddev": 0
+    }
+}
+----
+
+== Metadatos Métrica
```
Comment (willesreis) [inline examples/mp-metrics-timed/README_pt.adoc:148]:
```suggestion
== Metadados de Métrica
```

File: examples/mp-metrics-timed/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+Uma métrica terá metadados para que você possa aprender mais sobre ela, como `displayName`,`description`, `tags`, etc.
```
Comment (willesreis) [inline examples/mp-metrics-timed/README_pt.adoc:150]:
```suggestion
Uma métrica terá metadados na qual você pode saber mais sobre ele, como `displayName`,`description`, `tags`, etc.
```

**Comment by Daniel-Dos** [comment]
thanks @willesreis and @cesarhernandezgt  for review .

**Comment by Daniel-Dos** [comment]
thanks.

**Pull request merged**